packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -67,6 +67,10 @@ class AppSizeController {
   static const identicalFilesError =
       'Failed to load diff: OLD and NEW files are identical.';
 
+  static const artificialRootNodeName = 'ArtificialRoot';
+
+  static const mainNodeName = 'Main';
```

Comment:
we should use the functionality @elliette added in https://github.com/flutter/devtools/pull/4346 once that lands.